azure/azure-endpoints-availability/azure_connectivity_checker.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+
+# Azure endpoints to check
+endpoints=("https://management.azure.com/" "https://login.microsoftonline.com/" "https://management.core.windows.net/")
+isProxyUsed=false
+config_dir=""
+ssl_cert=""
+silent_mode=false
+
+# Create a folder named "azure_connectivity_logs" if it doesn't exist
+log_folder="/var/lib/dynatrace/gateway/temp/azure_connectivity_logs" || { echo "Error creating log folder."; exit 1; }
```

Comment:
this path assumes that the script is being executed in a computer with ActiveGate installed. I think that should be mentioned in the README

Reply:
I have this info at the very beginning of README. However, it might not be straightforward.
`This script is designed to test the connectivity to Azure endpoints from machine where ActiveGate is installed, both with and without proxy settings. `
Do you mean to add this info to the `Execute the Script` section too?